modules/packages/LinearAlgebra.chpl

```diff
@@ -1225,34 +1225,37 @@ module Sparse {
   pragma "no doc"
   /* Sparse-accumulator */
   record _SPA {
-    var cols: int;
+    var cols: range;
```

Comment:
IIRC, I think that this field will necessarily be a bounded, non-stridable range.  It might make sense to have it be a generic field in order to support domains that are stridable (unless we know for some reason that they will not be).  Really, we'd like to declare this field to be of type `range(?)` but I think at present, that only works in formal argument lists, not in field declarations (?).  I usually handle cases like this where I control the callsites by using a commented out type: `cols /* : range(?); */;`